c10/test/dispatch/OpSchema_test.cpp

```diff
@@ -1,26 +1,27 @@
-#include "c10/core/dispatch/OpSchema.h"
+#include <c10/core/dispatch/OpSchema.h>
 #include <c10/util/Array.h>
 
 using namespace c10;
-using namespace caffe2;
 
-static_assert(details::is_tensor_arg<Tensor>::value, "");
-static_assert(details::is_tensor_arg<const Tensor&>::value, "");
-static_assert(details::is_tensor_arg<Tensor&&>::value, "");
+static_assert(details::is_tensor_arg<C10Tensor>::value, "");
+static_assert(details::is_tensor_arg<const C10Tensor&>::value, "");
+static_assert(details::is_tensor_arg<C10Tensor&&>::value, "");
 static_assert(!details::is_tensor_arg<int>::value, "");
 
 struct SchemaDef final {
-  using Signature = bool(int, Tensor, float, Tensor, Tensor, unsigned int);
+  using Signature = bool(int, C10Tensor, float, C10Tensor, C10Tensor, unsigned int);
   static constexpr guts::array<const char*, 6> parameter_names = {{
       "1", "2", "3", "4", "5", "6"
   }};
+  static constexpr size_t num_dispatch_args() {return 3;}
+  static constexpr size_t num_outputs() {return 0;}
```

Comment:
So, what's going on here? Previously, I don't believe it was necessary to specify this information. This seems strictly worse than previously when it was inferred. Is there some usability issue that requires it to be specified explicitly now?